### PR TITLE
Allow resubmit of rejected application

### PR DESCRIPTION
**Pull request by compwron**
https://github.com/doubleunion/arooo/issues/295

### CODE REVIEW

File: app/models/application.rb [hunk 1]
```diff
@@ -91,6 +91,7 @@ def votes_threshold_email
 
     event :submit do
       transition started: :submitted
+      transition rejected: :submitted
```
Comment (Imperiopolis) [inline app/models/application.rb]: Do we possibly need to add another state here? I'm wondering if there is a valid case where we reject someone and we don't want them to be able to apply again. I am not familiar enough with the application process to know in what situations someone would be rejected.

**Comment by compwron** [comment]
Me neither. Who can we ask? I can ask in slack/ email?

**Comment by Imperiopolis** [comment]
> Me neither. Who can we ask? I can ask in slack/ email?

You can probably inquire in the app slack channel, yeah :)

File: app/models/application.rb (hunk 1 above)
Comment (brittag) [reply app/models/application.rb]: Based on our current [Voting Members Policy](https://docs.google.com/document/d/12R7utXAiyCK55XEP8cPscUqu2CXAj-hL5jXE-MaJdgE/edit#heading=h.y6eimvxrwzvf), there are no cases where a person should be technologically prevented from applying again. There are a few cases where specific people would be simply rejected if they applied again.

**Pull request merged**